File: app/src/main/java/com/digicoffer/lauditor/DocEditor/DocListingModel.java
package com.digicoffer.lauditor.DocEditor;

import org.json.JSONObject;

public class DocListingModel {
    String docid;
    String documentname;
    String $date;
    JSONObject updatedon;

    public String get$date() {
        return $date;
    }

    public void set$date(String $date) {
        this.$date = $date;
    }

    public String getDocid() {
        return docid;
    }

    public void setDocid(String docid) {
        this.docid = docid;
    }

    public String getDocumentname() {
        return documentname;
    }

    public void setDocumentname(String documentname) {
        this.documentname = documentname;
    }

    public JSONObject getUpdatedon() {
        return updatedon;
    }

    public void setUpdatedon(JSONObject updatedon) {
        this.updatedon = updatedon;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/DocEditor/DocListingModel.kt
package com.digicoffer.lauditor.DocEditor

import org.json.JSONObject

class DocListingModel {
    @JvmField
    var docid: String? = null
    
    @JvmField
    var documentname: String? = null
    
    @JvmField
    var date: String? = null
    
    @JvmField
    var updatedon: JSONObject? = null

    fun `get$date`(): String? = date
    
    fun `set$date`(value: String?) {
        date = value
    }

    fun getDocid(): String? = docid
    
    fun setDocid(value: String?) {
        docid = value
    }

    fun getDocumentname(): String? = documentname
    
    fun setDocumentname(value: String?) {
        documentname = value
    }

    fun getUpdatedon(): JSONObject? = updatedon
    
    fun setUpdatedon(value: JSONObject?) {
        updatedon = value
    }
}
